mor/mor/feel/feelmor/parameterspace.hpp

```diff
@@ -1584,7 +1667,8 @@ class ParameterSpace: public CommObject, public std::enable_shared_from_this<Par
         {
             M_min = min;
             M_min.setParameterSpace( this->shared_from_this() );
-
+            if( !checkMubar() )
```

Comment:
@romainhild I don't know if you followed the discussion in https://github.com/feelpp/feelpp/pull/1777#discussion_r811936212 with  @prudhomm : `mubar` should not be in the class `ParameterSpace` (we are working with @prudhomm to another way of configuring it)
If we revert this, have these tests to be present in function `setMin` and `setMax` ?

Reply:
nope